release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
+
+* Previously, the *Port* list in the *Add* flow only provided options for exposed ports and did not allow you to specify a custom port. The list has now been replaced by a typeahead select menu, and now it is possible to specify a custom port while creating the application. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881881[*BZ#1881881*])
+
+* Previously, when conditional tasks failed, the completed pipeline runs showed a permanent pending task for each failed conditional task. This issue has been fixed by disabling the failed conditional tasks and by adding skipped icons to them. This gives a better picture of the state of the pipeline run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880389[*BZ#1880389*])
+
+* Previously, the pod scale up or down buttons were available for a single pod resource, and the page crashed when the user pressed the scale button. This issue has been fixed by not showing the scale up or down buttons for a single pod resource. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909678[*BZ#1909678*])
+
+* Previously, the chart URL for downloading the chart to instantiate a helm release was unreachable. This happened because the `index.yaml` file from the remote repository, referenced in the Helm chart repository, was fetched and used as is. Some of these index files contained relative chart URLs. This issue has now been fixed by translating relative chart URLs to absolute URLs, which makes the chart URL reachable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1912907[*BZ#1912907*])
+
+* With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
+
+*DNS*
+
+* Previously, a cluster might experience intermittent DNS resolution errors because the `/etc/hosts` file on some nodes included invalid entries. With this release, DNS resolution no longer fails because of an `/etc/hosts` file with invalid entries. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882485[*BZ#1882485*])
+
+*etcd*
+
+* Previously, the etcd readiness probe used `lsof` and `grep` commands, which could leave zombie processes. The etcd readiness probe now uses a TCP port probe, which is less expensive and does not create zombie processes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1844727[*BZ#1844727*])
```

Comment:
Can we say "zombie processes"?

Reply:
zombie = defunct. I'll make the edit just in case :smile: